packages/pluggableWidgets/timeline-web/.gitignore

```diff
@@ -0,0 +1,4 @@
+dist/
```

Comment:
I don't think we should do this, parent `.gitignore` covers it

Reply:
done